src/containers/Main.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import {SortableContainer, SortableElement, arrayMove} from 'react-sortable-hoc';
+// import RaisedButton from 'material-ui/RaisedButton';
+import RaisedButtons from '../components/RaisedButton'
+// import './App.css';
+// import 'react-virtualized/styles.css'
+// Using an ES6 transpiler like Babel
+import PropTypes from 'prop-types';
+
+// Needed for onTouchTap, so error goes away
+import injectTapEventPlugin from 'react-tap-event-plugin';
+injectTapEventPlugin();
+
+import {connect} from 'react-redux'
+import {updateName, updateEngine} from '../actions'
+import RulesTable from '../components/RulesTable'
+
+class Main extends Component {
+  nameChangeHandler = (event) =>  {
+    const dispatch = this.props.dispatch
+    let act = updateName(event.target.value)
+    dispatch(act)
+  }
+  engineChangeHandler = (engineName) => {
+    // console.log('engineChangeHandler:', engineName)
+    // return // todo remove me
+
+    const dispatch = this.props.dispatch
+    let act = updateEngine(engineName)
+    dispatch(act)
+  }
+   render() {
+     return (
+     <div>
+       <h1>Hello World</h1>
+       <div>
+         <input value={this.props.name} onChange={this.nameChangeHandler}/>
```

Comment:
What is `name`? Is this just test code? If so it should also be removed from the state/reducer too.